packages/mkc-browser/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "mkc-browser",
```

Comment:
there is already an unrealated "mkc" package in npm, so we should stay away from this name

Reply:
https://www.npmjs.com/package/mkc